src/Workspaces/Core/Portable/Shared/Extensions/SyntaxGeneratorExtensions_CreateEqualsMethod.cs

```diff
@@ -382,14 +380,12 @@ private static bool ShouldUseEqualityOperator(ITypeSymbol typeSymbol)
         }
 
         private static ITypeSymbol GetType(Compilation compilation, ISymbol symbol)
-        {
-            switch (symbol)
+            => symbol switch
             {
-                case IFieldSymbol field: return field.GetTypeWithAnnotatedNullability();
-                case IPropertySymbol property: return property.GetTypeWithAnnotatedNullability();
-                default: return compilation.GetSpecialType(SpecialType.System_Object);
-            }
-        }
+                IFieldSymbol field => field.GetTypeWithAnnotatedNullability(),
+                IPropertySymbol property => property.GetTypeWithAnnotatedNullability(),
+                _ => compilation.GetSpecialType(SpecialType.System_Object),
+            };
```

Comment:
:memo: This change unnecessarily complicated this review.

Reply:
Broken windows.  Code that is out of spec with the repo guidelines end up cluttering error list and impacts the ability to keep it at zero items :)

Reply:
Just keep in mind it runs the risk that if I see too many unrelated changes in a PR, I just stop the review and then it sits forever. This one was close enough to the limit to leave the comments.

Reply:
As an external contributer you're in a damned-if-you-do, damned-if-you-don't position.  When i've tried sending through independent PRs to deal with all the IDE messages, i've been similarly blocked from making progress.  There seems to be no good way to satisfy all the differnet desires of the team.